sanic/types/shared_ctx.py

```diff
@@ -35,9 +35,12 @@ def _check(self, name: str, value: Any) -> None:
             module = value.__module__
         except AttributeError:
             module = ""
-        if not any(
-            module.startswith(prefix)
-            for prefix in ("multiprocessing", "ctypes")
+        if (
+            not any(
+                module.startswith(prefix)
+                for prefix in ("multiprocessing", "ctypes")
+            )
+            and not type(value) is bytes
```

Comment:
[why not `isinstance`?](https://stackoverflow.com/a/1549854)

Reply:
just didn't think about it while I was drinking my coffee - I'll update.